### ISSUE TITLE
Cannot read property 'startHrTime' of undefined

### ISSUE THREAD

**Issue by bingneef** [apollographql/apollo-tracing-js#6]
G'day!

When using tracing in combination with subscriptions (using pubsub), I get errors since `process` in undefined. Do you have any clue how to fix this? (for example ignore when coming from a socket?)

```
router.all('/graphql',
  (ctx, next) => graphqlKoa({
    schema,
    rootValue: {
      ctx,
    },
    tracing: true,
    cacheControl: true,
  })(ctx, next)
)

---------------------------------------

Subscription: {
    messageSubscription: {
      subscribe:
        withFilter(
          () => pubsub.asyncIterator('messageSubscription'),
          (payload, variables) => payload.messageSubscription.conversationId == variables.conversationId,
        )
    }
  },
```

**Comment by martijnwalraven** [comment]
Hmmm, what error message do you get? What do you mean by `process` is undefined?

**Comment by bingneef** [comment]
Hi Martijn!

This is my error message (in the errors of the GraphQL response): `Cannot read property 'startHrTime' of undefined`. 

Just realised that the issue is not with `process` (sorry, I misread that the property `startHrTime` is read from the `process` variable), however I was able to pinpoint it in the [index.js](https://github.com/apollographql/apollo-tracing-js/blob/master/src/index.ts) at line :58.

```
const resolverCall: ResolverCall = {
      path: info.path,
      fieldName: info.fieldName,
      parentType: info.parentType,
      returnType: info.returnType,
      startOffset: process.hrtime(this.startHrTime)
};
```

In the compiled version, `context._traceCollector` (`this` in the .ts) is undefined for subscriptions, and thus an error is thrown since it cannot read property startHrTime of `undefined`. A possible fix would be to just skip the tracing of subscriptions, but I don't know the impact of this. I could create a PR tomorrow if you'd like.

**Comment by martijnwalraven** [comment]
Ah, that is definitely something we should fix. I think it should be ok not to trace subscriptions for now, so a PR for that would be great!

**Comment by bingneef** [comment]
Dear Martijn,

I just realised that I was still using `apollo-server-koa` `1.1.2`, instead of `1.2.0` which used `0.0.7` and not the latest and greatest `0.1.1` which actually fixes the issue (no tracing on subscriptions). Figured this out when the build from the repo was significantly that from my `node_modules`.

tl;dr: sorry to have wasted your time and thank you for all the good work (y)
